internal/commands/push.go

```diff
@@ -91,7 +93,11 @@ func NewPushCommand(ctx context.Context, logger *log.Logger) *cobra.Command {
 			}
 
 			logger.Printf("pushing bundle to: %s", repository)
-			manifest, err := pushBundle(orascontext.Background(), repository, viper.GetString("policy"))
+			dataPath := viper.GetString("data")
+			if dataPath == "" {
+				dataPath = viper.GetString("policy")
```

Comment:
It might be more intuitive for the default to be "./data" but that could cause unexpected breakage for some users who aren't expecting their data to be included in the bundle. So hence I think probably this the best choice for a a default.

Reply:
Yeah i was thinking that as well. The other reason is current users may still be including data in the same directory as the policy. Defaulting to `./data` would mean it wouldn't pick up on those files.